### DISCUSSION TITLE
landscape透明化改进（红色系，莲华😋）

### DISCUSSION THREAD

**Discussion by Ytyo** [xiaoshuapp/effector#18] [Show and tell]
壁纸：
![294647413-b76abda9-9e77-4f48-9b20-69e4c37cfe33](https://github.com/xiaoshuapp/effector/assets/107429405/0ee02ea0-f44c-48bb-9b88-a1e966630922)

https://github.com/xiaoshuapp/effector/assets/107429405/4269ca60-b375-4b19-8d9b-f57f11985a84

.v-input__control{
 background-color: rgba(0, 0, 0, 8%) !important
}
/* 隐藏滚动条 */
::-webkit-scrollbar {
	display: none;
}

body {
	scrollbar-width: none;
}
.v-container {
	max-width:100%;
	padding-left:10rem;
	padding-right:5rem;
    margin-top: 80px
}
.drawer {
 background-color: rgba(205, 180, 180, 30%) !important
}
.bookmarks{
 background-color: rgba(205, 180, 180, 30%) !important;
}
.message{
    margin-top: 40px
}
.bookmarks{
 background-color: rgba(205, 180, 180, 30%) !important;
}
.flow .flow-header{
color: rgb(255, 255, 255);
font-size: 2.4em;
 font-weight: bold
}
.square-title.text-outer {	
color: rgb(255, 255, 255);
font-size: 1em;
 font-weight: bold
}
.shu-text-primary {	
color: rgb(80, 0, 0);
font-size: 1em;
 font-weight: bold
}
v-container v-locale--is-ltr cb{	
color: rgb(100, 0, 0);
font-size: 1em;
 font-weight: bold
}
.message[data-v-dc746abd] {
font-size: 3em;
 font-weight: bold;
    margin-top: 60px}
.v-navigation-drawer--temporary{ background-color: rgba(205, 0, 0, 30%) !important;
}
.v-navigation-drawer--floating{
background-color: rgba(205, 0, 0, 30%) !important;
}
.v-navigation-drawer--left {
background-color: rgba(205, 0, 0, 30%) !important;
}
.v-navigation-drawer__content{
color: rgb(255, 255, 255);
font-size: 2.4em;
 font-weight: bold
}
.v-theme--light{
color: rgb(100, 10, 10,1);
 font-weight: bold
}

.v-application{
background-color: rgba(205, 0, 0, 100%) !important;
}

.bookmarks {
background-color: rgba(155, 0, 0, 0%) !important;
}
.flow-header .v-btn{
background-color: rgba(255, 255, 255, 50%) !important;
}
.flow-header{
background-color: rgba(180, 10, 10, 5%) !important;
}
.Landscape .space{
 margin-top: 470px
}
.btn[data-v-33a079d0] {
    width: 32px;
    height: 32px;
    color: inherit;
    border-radius: 50%!important;
    transition: all .1s ease-in-out;
    margin-right: .75rem;
    display: none;
}
}

**Comment by nexmoe** [comment]
谨慎使用 data-v- 这样类似的东西
